cow_amm/uni_v2_amm_lp_and_tvl_4047194

```diff
@@ -0,0 +1,94 @@
+WITH 
+cow_amm_pool as (
+select blockchain, address, token_1_address, token_2_address
+from query_3959044 q
+where address={{cow_amm_pool}}
+)
+
+, uniswap_pool as 
+    (select output_0 as pool from uniswap_v2_ethereum.Factory_call_getPair
+    where ((_0 in (select token_1_address from cow_amm_pool) and _1 in (select token_2_address from cow_amm_pool) )
+    or (_0 in (select token_2_address from cow_amm_pool) and _1 in (select token_1_address from cow_amm_pool)))
+    and not(output_0 = 0x0000000000000000000000000000000000000000)
+    limit 1
+    )
+    
+, get_lp_balance as (
```

Comment:
I would maybe rename this a bit. If i understand correctly, this computes how many LP tokens were minted and burned on a given day, right? So maybe something very literal as `lp_token_mint_burn_per_day`